Makefile

```
@@ -48,6 +48,9 @@ pmm2-ovf: fetch
 pmm2-ami:
 	packer build -only amazon-ebs packer/pmm2.json
 
+pmm2-az:
```

```suggestion
pmm2-azure:
```
To avoid ambiguity between `azure` and Availability Zone` which is also `az` )

---

Changed!